ldes-fragmentisers/ldes-fragmentisers-substring/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/substring/SubstringFragmentationStrategyTest.java

```diff
@@ -45,20 +37,18 @@ class SubstringFragmentationStrategyTest {
 	void setUp() {
 		PARENT_FRAGMENT = new LdesFragment(
 				VIEW_NAME, List.of());
-		localMemberSupplier = mock(LocalMemberSupplier.class);
 		substringFragmentFinder = mock(SubstringFragmentFinder.class);
 		substringFragmentCreator = mock(SubstringFragmentCreator.class);
+		substringConfig = mock(SubstringConfig.class);
```

Comment:
Mocking a simple POJO can be considered a test smell. Since this class contains no logic, you don't really have to mock it.